uspace/lib/c/arch/ppc32/src/fibril.S

```diff
@@ -89,13 +89,13 @@ FUNCTION_BEGIN(context_restore)
 	lwz r30, CONTEXT_OFFSET_R30(r3)
 	lwz r31, CONTEXT_OFFSET_R31(r3)
 
-	lwz r4, CONTEXT_OFFSET_CR(r3)
-	mtcr r4
+	lwz r5, CONTEXT_OFFSET_CR(r3)
+	mtcr r5
 
-	lwz r4, CONTEXT_OFFSET_PC(r3)
-	mtlr r4
+	lwz r5, CONTEXT_OFFSET_PC(r3)
+	mtlr r5
 
-	# context_restore returns 0
-	li r3, 0
+	# __longjmp returns second argument
+	li r3, r4
```

Comment:
This is probably wrong. `li` stands for load immediate?

Reply:
Yeah, very likely. Can't you just mov or move? In the worst case, RTFM :-)